components/map/overview.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers.
+
+The component provides tile layers, shape (vector) layers, bubble layers, and marker layers.
```

Comment:
use slugs for the listed items. They all have their own slugs, so consider linking them here.